### ISSUE TITLE
 command: storage-node leader:update-bag Channels range

### ISSUE THREAD

**Issue by yasiryagi** [Joystream/joystream#4591]
It would be helpful if the command below would accept channels range or comma separated set

storage-node leader:update-bag

Example 

Current:
yarn storage-node leader:update-bag -i dynamic:channel:7733 -a 3
yarn storage-node leader:update-bag -i dynamic:channel:7734 -a 3
yarn storage-node leader:update-bag -i dynamic:channel:7735 -a 3

Request:
yarn storage-node leader:update-bag -i dynamic:channel:7733 dynamic:channel7735 -a 3
or 
yarn storage-node leader:update-bag -i dynamic:channel:7733,dynamic:channel7734,dynamic:channel7735 -a 3

**Comment by mnaamani** [comment]
This should be straightforward to add, and does not require a runtime change to the underlying dispatch call.

The command of uses the `api.tx.storage.updateStorageBucketsForBag` dispatch, which operates on a single bag. It can fail for multiple reasons:

```
DynamicBagDoesntExist
StorageBucketPerBagLimitExceeded
StorageBucketDoesntExist
StorageBucketIsNotBoundToBag
StorageBucketDoesntAcceptNewBags
StorageBucketIsBoundToBag
StorageBucketObjectNumberLimitReached
StorageBucketObjectSizeLimitReached
```

We can implement one or both strategies below lead the choice depending on desired outcome.

1. Single Extrinsic: batching multiple calls using `api.tx.utils.batch()` dispatch. This approach is useful if the update operation on the bags need to be atomic (either all bags are updated or none are updated) There is more chance this will fail if the lead doesn't analyze the current state of buckets.
1. Multiple Extrinsics: one Extrinsic sent per bag update operation. The optimum outcome is all bags are updated successfully as desired, but the user must expect that only a subset of the bags will be updated.  For speed the extrinsics shoud be sent out in parallel (which nonce increments).. otherwise this is no better than a bash script invoking the cli commands sequentially.

When implementing
  - do some input validation to help prevent failed transaction
  - remove duplicate bag ids in input
  - update the argument to allow `multiple` so the bagId argument can be passed multiple times. usage becomes:
  ```sh
yarn storage-node leader:update-bag -i dynamic:channel:7733 -i dynamic:channel7735 -a 3
  ```
  - Pluralize the the command name to `update-bags`
  - Add an argument to specify which TX sending strategy to use.

**Comment by zeeshanakram3** [comment]
There are there `batch` variants available in the utility pallet of substrate

- [batch](https://github.com/paritytech/substrate/blob/master/frame/utility/src/lib.rs#L164): Would succeed up to `n` successful calls 
- [batch_all](https://github.com/paritytech/substrate/blob/master/frame/utility/src/lib.rs#L291): Either all calls would succeed on none
- [force_batch](https://github.com/paritytech/substrate/blob/master/frame/utility/src/lib.rs#L400): Will execute all calls even if there are error


I think we want to include both strategies for this action, then we can use `batch_all` for an atomic operation, and `force_batch` the errors are acceptable in batch.

**Comment by mnaamani** [comment]
> I think we want to include both strategies for this action, then we can use `batch_all` for an atomic operation, and `force_batch` the errors are acceptable in batch.

Perfect.

**Comment by zeeshanakram3** [comment]
Addressed in #4812
